lib/wsdl.js

```diff
@@ -807,17 +811,41 @@ EnumerationElement.prototype.description = function() {
 
 ComplexTypeElement.prototype.description = function(definitions, xmlns) {
   var children = this.children || [];
+  
+  var hasAttributes = false;
+  var isFirstChild = false;
+  var ret = {};
+ 
+  var $attributes = {};
+
   for (var i=0, child; child=children[i]; i++) {
-    if (child instanceof ChoiceElement ||
+
+    if( child instanceof AttributeElement){
+      hasAttributes = true;
+      $attributes[child.$name] = child.$type;
+      continue;
+    }
+
+    if (!isFirstChild && (child instanceof ChoiceElement ||
       child instanceof SequenceElement ||
       child instanceof AllElement ||
       child instanceof SimpleContentElement ||
-      child instanceof ComplexContentElement) {
-
-      return child.description(definitions, xmlns);
+      child instanceof ComplexContentElement))
+    {
+      isFirstChild = true;
+      ret = child.description(definitions, xmlns);
     }
   }
-  return {};
+
+  if(hasAttributes){
+    Object.defineProperty(ret, '$attributes',{
+      value: $attributes,
+      enumerable: false,
```

Comment:
Why would we want this to not be enumerable and therefore not appearing when calling `Object.keys()`, `Object.values()` and similar on this property?

Reply:
I wanted to limit as much as possible side-effects in order to manage attributes just in that piece of code.

Reply:
I'd vote for the property to be `enumerable` so that the fellow developer is able to read it within a loop or something the like. What about you @jsdevel ?

Reply:
@nastassja90 would you please update your PR and make it `enumerable`? this would be awesome 👍